src/test/java/study/step2/StringCalculatorTest.java

```diff
@@ -0,0 +1,64 @@
+package study.step2;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+class StringCalculatorTest {
+
+    private StringCalculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new StringCalculator();
+    }
+
+    @DisplayName("사칙연산 입력, null")
+    @Test
+    void inputNull() {
+        String input = null;
+        assertThatIllegalArgumentException().isThrownBy(() -> {
+            calculator.input(input);
+        });
```

Comment:
```suggestion
        assertThatIllegalArgumentException().isThrownBy(() -> {
            calculator.input(input);
        }).withMessage("error message");
```
withMessage 를 이용해 에러 메세지에 대한 테스트도 해주면 좋을것 같아요! 😄